src/Tribe/REST/V1/Endpoints/Archive_Event.php

```diff
@@ -97,6 +102,10 @@ public function get( WP_REST_Request $request ) {
 			$args['posts_per_page'] = get_option( 'posts_per_page' );
 		}
 
+		if ( current_user_can( 'edit_posts' ) ) {
```

Comment:
Shouldn't we use our own Cap? (from the Post Type Object?) Instead of fixed string.